config/routes.rb

```diff
@@ -1,13 +1,11 @@
 Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
   devise_for :users,
```

Comment:
Good work on creating the API documentation! :+1: 
![Screenshot from 2022-09-23 15-44-50](https://user-images.githubusercontent.com/97834160/191984609-8d80ac8a-34f4-4785-9992-f42364069e5f.png)